docker/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -xeuo pipefail
+set -euo pipefail
```

Comment:
-x is way too verbose for default config .. even the current setup still is rather loud..